test/uk/gov/hmrc/teamsandrepositories/models/RepoTypeSpec.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.teamsandrepositories.models
+
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.util.Random.nextInt
+
+object Helper extends App {
+
+  def randomly_capitalize(str: String): String  = {
+    str.map(char => nextInt() % 2 == 0 match {
+      case true => char.toUpper
+      case false => char.toLower
+    })
+  }
+
+  def generate_valid_strings(strs: Seq[String]): List[String] = {
```

Comment:
minor thing but scala generally uses camelCase (caps between words) rather than snake_case (underscores between word) for function names